docs/2.0/integration/testng.adoc

```diff
@@ -19,7 +19,7 @@ You need to add the following to your **pom.xml**:
     <compiler.plugin.version>3.6.0</compiler.plugin.version>
     <java.version>1.8</java.version>
     <surefire.plugin.version>2.19.1</surefire.plugin.version>
-    <allure.version>2.0-SNAPSHOT</allure.version>
+    <allure.version>2.0-BETA6</allure.version>
```

Comment:
The link above refers to allure-2.0.1. Here we have 2.0-BETA6. Quite a confusing discrepancy.